packages/compare-slider-block/src/components/Label/Label.tsx

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { RichTextEditor, parseRawValue, serializeRawToHtml, toPlaintext } from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import { useMemo } from 'react';
+import { labelPlugins } from '../../helpers';
+import { LabelProps } from '../../types';
+
+export const Label = ({ designTokens, onBlur, value, isEditing }: LabelProps) => {
+    const rawValue = useMemo(() => JSON.stringify(parseRawValue({ raw: value ?? '' })), [value]);
+    const valueHtml = useMemo(() => serializeRawToHtml(rawValue, designTokens), [rawValue, designTokens]);
+    const isEditorEmpty = !toPlaintext(rawValue);
+    const richTextEditor = useMemo(() => {
+        return (
+            <RichTextEditor
+                value={value}
+                onBlur={onBlur}
+                border={false}
+                designTokens={designTokens}
+                plugins={labelPlugins}
+                placeholder="Label content"
+            />
+        );
+    }, [value, onBlur, designTokens]);
+    return isEditorEmpty && !isEditing ? (
+        <div />
```

Comment:
Where opens that div? Seems a bit odd to me

Reply:
It is a self closing element. Shorthand for <div></div>

Reply:
Ah damn, yeah for sure :D